api/tests/routes/backoffice/venues_test.py

```diff
@@ -984,15 +986,12 @@ def test_update_venue_location_with_offerer_address_not_manual(self, authenticat
         assert response.location == url_for("backoffice_web.venue.get", venue_id=venue.id, _external=True)
 
         db.session.refresh(venue)
-        offerer_address = offerers_models.OffererAddress.query.one()
-        addresses = geography_models.Address.query.all()
+        offerer_address = offerers_models.OffererAddress.query.order_by(
+            offerers_models.OffererAddress.id.desc()
+        ).first()
+        assert (len(offerers_models.OffererAddress.query.all())) == 2
```

Comment:
Pas besoin de faire deux requêtes
```suggestion
        all_offerer_address = offerers_models.OffererAddress.query.all()
        assert len(all_offerer_address) == 2
        offerer_address = all_offerer_address[0]
```